apps/foundation-app/src/app/cart/components/cart/cart.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="cart-container">
+  {{carts[0].id}}
```

Comment:
Carts shouldn't be an array.

Reply:
At some point I think `carts` needs to be an array, because of how ngrx-entity works. It returns an array of whatever it stores on state, right? I just wasn't sure where it should be changed into an object. I could do it in the `cart.component` module instead:

```
this.carts$ = this.store.pipe(
      select(fromCart.selectAllCarts)
);
```
would change to 
```
this.cart$ = this.store.pipe(
      select(fromCart.selectAllCarts)[0]
);
```

Or maybe I just shouldn't be using ngrx-entity in this case?

Reply:
changed